packages/PlanLimitsUI/src/App.tsx

```diff
@@ -2,32 +2,24 @@ import {
   createBrowserRouter,
   RouteObject,
   RouterProvider,
-  redirect,
+  Navigate,
 } from 'react-router-dom';
 import Layout from './Layout';
-import Limits, { defaultViewLocation, loader } from './pages/Limits';
-import { createLocationString } from './pages/Limits/locationString';
+import Limits, { defaultPath, loader } from './pages/Limits';
 
 export const routes: RouteObject[] = [
   {
-    path: '/',
     element: <Layout />,
     children: [
-      {
-        index: true,
-        loader: () =>
-          redirect(`/limits/${createLocationString(defaultViewLocation)}`),
-      },
-      {
-        path: '/limits',
-        loader: () =>
-          redirect(`/limits/${createLocationString(defaultViewLocation)}`),
-      },
       {
         path: '/limits/:location',
         loader,
         element: <Limits />,
       },
+      {
+        path: '/limits',
+        element: <Navigate replace to={defaultPath} />,
+      },
```

Comment:
I think this simplifies our route config and removes the redundant loader needed to wrap the redirect.

Reply:
Nice, yeah thats much better